Source/Diff.swift

```diff
@@ -26,7 +26,7 @@ public enum Operation: Equatable {
     case let (.insert(l), .insert(r)),
          let (.delete(l), .delete(r)),
          let (.update(l), .update(r)): return l == r
-    case let (.move(l), .move(r)): return l == r
+    case let (.move(l1,l2), .move(r1,r2)): return l1 == r1 && l2 == r2
```

Comment:
Is this change related to an Xcode warning?

Reply:
Yes, seems like a new Swift 5.3 warning about decomposing tuples.